vernac/g_vernac.mlg

```diff
@@ -502,7 +502,7 @@ GRAMMAR EXTEND Gram
     [ [ p = lconstr -> { p } ] ];
   rewrite_rule:
     [ [ u = OPT [ u = univ_decl; "|-" -> { u }];
-        lhs = rw_pattern; "==>" ; rhs = lconstr ->
+        lhs = rw_pattern; ">->" ; rhs = lconstr ->
```

Comment:
Won't this break existing user code that relies on this?  Please address that--either explain why that isn't the case or describe the deprecation/compatibility strategy in the PR.

Reply:
Ah, this was just added, so not an issue?

Reply:
>Ah, this was just added, so not an issue?

yes